rails_programming/rails_basics/importmap.md

```diff
@@ -33,30 +37,30 @@ Import maps are essentially string substitution for "bare module specifiers". If
 * HTTP path:
     `import React from "https://ga.jspm.io/npm:react@17.0.1/index.js"`
 
-The `importmap-rails` gem provides an API for mapping the "bare module specifiers" to 1 of the 3 ways of loading ESM JavaScript packages. 
+The `importmap-rails` gem provides an API for mapping the "bare module specifiers" to 1 of the 3 ways of loading ESM JavaScript packages.
+
+Example:
 
-Example: 
-```Ruby
+~~~Ruby
 # config/importmap.rb
 pin "react", to: "https://ga.jspm.io/npm:react@17.0.2/index.js"
-```
-This means "every time you see `import React from "react"` in the code change it to `import React from "https://ga.jspm.io/npm:react@17.0.2/index.js"`"
+~~~
 
-Without the import maps gem, you would have to manually go through your code and update the import statements to have each new digest name when your code updates.
+This means "every time you see `import React from "react"` in the code it will actually be like writing `import React from "https://ga.jspm.io/npm:react@17.0.2/index.js"`"
```

Comment:
```suggestion
This means that every time you see `import React from "react"` in the code it will actually be like writing `import React from "https://ga.jspm.io/npm:react@17.0.2/index.js"`.
```